pakyow-presenter/lib/pakyow/presenter/presentable.rb

```diff
@@ -8,13 +8,27 @@ def initialize(*args)
         super
       end
 
-      def presentable(*args)
-        presentables.concat(args).uniq!
+      def presentable(name, value = nil)
+        args = {
```

Comment:
I think this was a bug in my original implementation, but `presentables` should probably be a hash with keys being the name of the presentable. That way we don't have to uniq anything; values will naturally override.